examples/listener/example1.tf

```diff
@@ -0,0 +1,50 @@
+module "webapp" {
+  source     = "USSBA/easy-fargate-service/aws"
+  version    = "~> 4.0"
+  depends_on = [module.efs]
+
+  # logging
+  log_group_name              = "/ecs/${local.env.name}/webapp"
+  log_group_retention_in_days = 90
+
+  # task
+  family      = "${local.env.name}-webapp"
+  task_cpu    = 2048
+  task_memory = 4096
+
+  # scaling
+  max_capacity = 10
+  min_capacity = 2
+
+  deployment_maximum_percent         = 200
+  deployment_minimum_healthy_percent = 100
+
+  # networking
+  service_fqdn       = local.env.webapp_fqdn
+  hosted_zone_id     = local.env.hosted_zone_id
+  private_subnet_ids = module.vpc.private_subnets
+  vpc_id             = module.vpc.vpc_id
+  listeners          = [{ port = 8080, protocol = "HTTP", action = { type = "forward" } }]
+
+  # container(s)
+  cluster_name   = aws_ecs_cluster.this.name
+  container_port = 8080
+  container_definitions = [
+    {
+      name         = "webapp"
+      portMappings = [{ containerPort = 8080 }]
+      image        = "${local.prefix_ecr}/webapp:${local.env.webapp_image_tag}"
+    }
+  ]
+
+  # efs configuration(s)
+  efs_configs = [
```

Comment:
I'd strip out a lot of this extra stuff.  No need for EFS mappings just to demo how this works.  For the listener directory, just have the listeners, and then as little other config as you can get away with and have it still make sense